doc/en/components/date-picker.md

```diff
@@ -0,0 +1,90 @@
+---
+title: $PlatformShort$ Date Picker | Data Visualization Tools | Infragistics
+_description: Infragistics' $PlatformShort$ date picker component helps your users select dates. Improve your graphs and visualization with Ignite UI for  $PlatformShort$!
+_keywords: $PlatformShort$ date picker, drop down, $ProductName$, Infragistics
+mentionedTypes: []
```

Comment:
in the `mentionedTypes` array, add name of class(es) that have implementation of properties mentioned in this topic